ols/app/endpoints/authorized.py

```diff
@@ -5,15 +5,39 @@
 
 from fastapi import APIRouter, Request
 
-from ols.app.models.models import AuthorizationResponse
+from ols.app.models.models import (
+    AuthorizationResponse,
+    ErrorResponse,
+    ForbiddenResponse,
+    UnauthorizedResponse,
+)
 from ols.utils.auth_dependency import AuthDependency
 
 logger = logging.getLogger(__name__)
 router = APIRouter(tags=["authorized"])
 auth_dependency = AuthDependency(virtual_path="/ols-access")
 
-
-@router.post("/authorized")
+authorized_responses = {
+    200: {
+        "description": "The user is logged-in and authorized to access OLS",
+        "model": AuthorizationResponse,
+    },
+    401: {
+        "description": "Missing or invalid credentials provided by client",
+        "model": UnauthorizedResponse,
+    },
+    403: {
+        "description": "User is not authorized",
+        "model": ForbiddenResponse,
+    },
+    500: {
+        "description": "Unexpected error during token review",
+        "model": ErrorResponse,
+    },
+}
+
+
+@router.post("/authorized", responses=authorized_responses)
```

Comment:
does this affect behavior, or just the openapi schema generation?

Reply:
it affects schema and also Swagger UI (http://localhost:8080/docs), nothing else